src/main/resources/org/csanchez/jenkins/plugins/kubernetes/volumes/workspace/DynamicPVCWorkspaceVolume/config.jelly

```diff
@@ -0,0 +1,16 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+
+    <f:entry title="${%StorageClass Name}" field="storageClassName">
+        <f:textbox />
+    </f:entry>
+
+    <f:entry title="${%Requests Size}" field="requestsSize">
+        <f:textbox />
+    </f:entry>
+
+    <f:entry title="${%Access Modes}" field="accessModes">
```

Comment:
Is there even a use case for something else than `ReadWriteOnce` ? Since the pvc is dynamic, it won't be shared with other clients. If this is kept, using a [combobox](https://github.com/jenkinsci/jenkins/blob/cfbf4ca45ec97c05b47b69d1d59d3f5ae1f50c82/core/src/main/resources/lib/form/combobox.jelly#L29) with a fill method would provide a better UX.

Reply:
In the case of dynamic storage volumes, there will be more use of the ReadWriteOnce mode, but there may be other modules that need to mount this PVC, such as watch jenkins agent and create another pod to record changes in the pv workspace.